packages/firestore/test/integration/util/helpers.ts

```diff
@@ -191,18 +191,25 @@ export function withTestDbsSettings(
   }
 
   return Promise.all(promises).then((dbs: firestore.Firestore[]) => {
-    return fn(dbs)
-      .then(wipeDb.bind(null, dbs[0]), error => {
-        return wipeDb(dbs[0]).then(() => {
-          throw error;
-        });
-      })
-      .then(() => {
-        return dbs.reduce(
+    const cleanup = () => {
+      return wipeDb(dbs[0]).then(() =>
```

Comment:
Why don't we call wipeDb for every database as part of dbs.reduce? The extra overhead of calling a function that doesn't do anything should be fairly small :)

Reply:
wipeDb is meant to delete the contents of the database.  Since all of the db objects are just separate Firestore instances that point to the same underlying database, there's no point in doing the wipe multiple times. The most correct thing would probably actually be to create a new Firestore instance (rather than use dbs[0] which may be in a broken / unknown state after the test runs), but given wipeDb() isn't implemented I don't care. :-)